app/src/main/java/com/googleresearch/capturesync/softwaresync/phasealign/PeriodCalculator.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Modifications copyright (C) 2021 Mobile Robotics Lab. at Skoltech.
+ */
+
+package com.googleresearch.capturesync.softwaresync.phasealign;
+
+import android.content.Context;
+import android.os.Build;
+
+import androidx.annotation.RequiresApi;
+
+import net.sourceforge.opencamera.R;
+import net.sourceforge.opencamera.ToastBoxer;
+import net.sourceforge.opencamera.preview.Preview;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class PeriodCalculator {
+    private final static long CALC_DURATION_MS = 10000;
```

Comment:
Maybe we should put this into config file (as some other recsync params), I made it as more of a temporary solution

Reply:
The class with such params is SyncConstants, I think

Reply:
Speaking about this, there are some constants like this in `PhaseAlignController` [here](https://github.com/MobileRoboticsSkoltech/OpenCamera-Sensors/blob/9415fb2a49adc9b57f6f4e1b3da736b23bec3c5f/app/src/main/java/com/googleresearch/capturesync/PhaseAlignController.java#L54) -- should I move them to `SyncConstants` too?